CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## V0.12.00
```

Comment:
Maybe I missed something but we are not releasing the v0.12.0 but the v0.11.1

Reply:
```suggestion
## V0.11.1
```